time/src/test/java/net/jqwik/time/api/LocalTimeTests.java

```diff
@@ -482,13 +483,12 @@ void defaultShrinking(@ForAll Random random) {
 			assertThat(value).isEqualTo(LocalTime.of(0, 0, 0, 0));
 		}
 
-		//TODO: Too much time?
 		@Property
 		void shrinksToSmallestFailingValue(@ForAll Random random) {
-			LocalTimeArbitrary times = Times.times();
+			LocalTimeArbitrary times = Times.times().constrainPrecision(SECONDS);
 			TestingFalsifier<LocalTime> falsifier = time -> time.isBefore(LocalTime.of(9, 13, 42, 143_921_111));
 			LocalTime value = falsifyThenShrink(times, random, falsifier);
-			assertThat(value).isEqualTo(LocalTime.of(9, 13, 42, 143_921_111));
+			assertThat(value).isEqualTo(LocalTime.of(9, 13, 43, 0));
```

Comment:
If the testcase works it still needs 6 seconds. 
If it fails it needs only 60 milliseconds. Why is that?
Example:
```
Shrinking:shrinksToSmallestFailingValue = 
  org.opentest4j.AssertionFailedError:
    Expecting:
     <09:13:43>
    to be equal to:
     <09:13:42>
    but was not.
```